pkg/commands/version.go

```diff
@@ -26,65 +24,75 @@ type versionInfo struct {
 	BuildInfo *debug.BuildInfo
 }
 
-func (e *Executor) initVersion() {
+type versionOptions struct {
+	Format string `mapstructure:"format"`
```

Comment:
don't see these fields in config reference 🤔 
or what reason of mapstructrure?

Reply:
copy-pasting :smile:

Reply:
Removed in the next PR